awscli/__init__.py

```diff
@@ -15,6 +15,8 @@
 ----
 A Universal Command Line Environment for Amazon Web Services.
 """
+
```

Comment:
I'm pretty sure PEP8 doesn't require spaces between module level comments and import statements, though I could be wrong.